Makefile.am

```diff
@@ -174,6 +174,7 @@ release-and-tag:
 	$(MAKE) tag
 
 anaconda-ci-build:
+	cp $(srcdir)/anaconda.spec.in $(CI_DOCKERFILE)/anaconda.spec.in
```

Comment:
So, I will have multiple copies of the `anaconda.spec.in` file all over my git repo, right? That is just sad. Shouldn't we start to create a temporary directory for the build context? Or could we use a build argument to propagate the content of the spec file?

Reply:
Maybe we can just remove the spec file after building, a few lines below.

Reply:
> So, I will have multiple copies of the `anaconda.spec.in` file all over my git repo, right? That is just sad. 

The file is in `.gitignore` so shouldn't really have to care. However, yes, we can definitely remove the file after the build finished. That is pretty easy to do, if you want.
Just a note that until recently we had a lot of files in our repo directory because of autogen and configure calls.

> Shouldn't we start to create a temporary directory for the build context?
I really don't want to complicate the Makefile by creating a temp dir for a build and copying everything there. It will be fragile and much more complex than what  we have now.
 
> Or could we use a build argument to propagate the content of the spec file?
Nope, the spec file is too big to pass the content as argument. We would probably get `line too long` from shell and it's just a wrong way of doing things in general.

Reply:
> > So, I will have multiple copies of the `anaconda.spec.in` file all over my git repo, right? That is just sad.
> 
> The file is in `.gitignore` so shouldn't really have to care. However, yes, we can definitely remove the file after the build finished. That is pretty easy to do, if you want.

I very often search my repository. It will appear there. The version of these files doesn't even have to match the branch, that I am using, so it should be at least removed.

> Just a note that until recently we had a lot of files in our repo directory because of autogen and configure calls.

Yes and I am very happy with the current state, so why do we want to change that?

> 
> > Shouldn't we start to create a temporary directory for the build context?
>
> I really don't want to complicate the Makefile by creating a temp dir for a build and copying everything there. It will be fragile and much more complex than what  we have now.

It would be cleaner and not so confusing as the current solution.

Reply:
> Honestly, I think creating the tmp dir and putting everything there would be more confusing. First it's making our Makefile bigger and harder to orient and second it will be harder to find the issue if something went wrong.

You need to copy there only the content of the `$(CI_DOCKERFILE)` directory and the spec file.

> So do I understand it correctly that you are fine with solution when the spec file will be removed after build?

Yes, it is ugly, but it will work.

Reply:
For the record - I would be happier with a better solution, too, but I would take this too. If I understand this correctly, the problem being solved affects 2/6 team members, potentially for the next month or so, so that makes me feel something should happen.